llmfoundry/data/finetuning/tasks.py

```diff
@@ -809,12 +826,15 @@ def dataset_mapper(example: Dict):
                 desc='Tokenizing dataset',
             )
 
-            filtered_dataset = tokenized_dataset.filter(
-                partial(is_valid_ift_example, max_seq_len, target_prompts,
-                        target_responses, decoder_only_format),
-                num_proc=num_cpus_to_use,
-                desc='Filtering out long prompts',
-            )
+            filter_timeout = 600
```

Comment:
can we make this configurable with a default? Also would suggest longer than 10 minutes, especially if it is not configurable. maybe 30

Reply:
Updated